src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -319,15 +350,66 @@ Note that variable names must be explicitly set and be lowercase like `%{URI:uri
     my_attribute_prefix.request.headers.X-Custom: "bar"
     ```
 
-    Optionally, you can configure the `json` [grok type](#grok-syntax) using `:json(_CONFIG_)`:
+    You can also configure the `json` [Grok type](#grok-syntax) using `:json(_CONFIG_)`:
 
     - `json({"dropOriginal": true})`: Drop the JSON snippet that was used in parsing. When set to `true` (default value), the parsing rule will drop the original JSON snippet. Note the JSON attributes will remain in the message field.
     - `json({"dropOriginal": false})`: This will show the JSON payload that was extracted. When set to `false`, the full JSON-only payload will be displayed under an attribute named in `my_attribute_prefix` above. Note the JSON attributes will remain in the message field here as well giving the user 3 different views of the JSON data. If storage of all three versions is a concern it's recommended to use the default of `true` here.
     - `json({"depth": 62})`: Levels of depth you want to parse the JSON value (defaulted to 62).
   </Collapser>
+  <Collapser
+    id="parsing-csv"
+    title="Parsing CSV"
+  >
+    If your system sends comma-separated values (CSV) logs and you need to parse them in New Relic, you can use the `csv` [Grok type](#grok-syntax), which parses the CSV captured by the Grok pattern.
+    This format relies on 3 main parts: the Grok syntax, the prefix you would like to assign to the parsed CSV attributes, and the `csv` [Grok type](#grok-syntax). Using the `csv` [Grok type](#grok-syntax), you can extract and parse CSV from logs.
+
+    Given the following CSV log line as an example:
+
+    ```
+    "2015-05-13T23:39:43.945958Z,202,POST,/shopcart/checkout,142,10"
+    ```
+
+    And a parsing rule with the following shape:
+
+    ```
+    %{GREEDYDATA:log:csv({"columns": ["timestamp", "status", "method", "url", "time", "bytes"]})
+    ```
+
+    Will parse your log as follows:
+
+    ```
+    log.timestamp: "2015-05-13T23:39:43.945958Z"
+    log.status: "202"
+    log.method: "POST"
+    log.url: "/shopcart/checkout"
+    log.time: "142"
+    log.bytes: "10"
+    ```
+
+    If you need to omit the "log" prefix, you can include the `"noPrefix": true` in the configuration.
+
+    ```
+    %{GREEDYDATA:log:csv({"columns": ["timestamp", "status", "method", "url", "time", "bytes"], "noPrefix": true})
+    ```
+
+    ### Columns configuration:
+
+    - It's mandatory to indicate the columns in the CSV Grok type configuration (which should be a valid JSON).
+    - You can ignore any column by setting "_" (underscore) as the column name to drop it from the resulting object.
+
+    ### Optional configuration options:
+
+    While the columns configuration is mandatory, there are other optional configurations that could help you in some cases:
```

Comment:
"that could help you" is a little vague, it would be helpful to have a small bit on what it could help with specifically

Reply:
Updated to include a "generic" but more accurate introduction for the list.